ios-base/Onboarding/Routes/OnboardingRoutes.swift

```diff
@@ -26,17 +26,13 @@ enum OnboardingRoutes: Route {
   }
 
   private func buildSignInViewController() -> UIViewController {
-    guard let signIn = R.storyboard.main.signInViewController() else {
-      return UIViewController()
-    }
+    let signIn = SignInViewController()
     signIn.viewModel = SignInViewModelWithCredentials()
     return signIn
   }
 
   private func buildSignUpViewController() -> UIViewController {
-    guard let signUp = R.storyboard.main.signUpViewController() else {
-      return UIViewController()
-    }
+    let signUp = SignUpViewController()
     signUp.viewModel = SignUpViewModelWithEmail()
```

Comment:
same here, we should now inject the viewmodels into the view controllers via the initalizer